setup.py

```diff
@@ -11,6 +11,7 @@
 
 setup(name='bluerobotics-ping',
       version='0.1.0',
+      python_requires='>=3',
```

Comment:
Maybe we should specify Python 3.5, since it was when type hinting was added.

Reply:
I was wondering what was the correct version, thanks!
this means it will be incompatible with the current companion image, though.